cli/src/cmd/forge/node.rs

```diff
@@ -0,0 +1,112 @@
+use clap::Parser;
+use ethers::{
+    core::{k256::ecdsa::SigningKey, types::U256},
+    prelude::{coins_bip39::English, MnemonicBuilder, Wallet},
+};
+
+use crate::opts::evm::EvmArgs;
+use foundry_node::NodeConfig;
+use std::str::FromStr;
+
+#[derive(Clone, Debug, Parser)]
+pub struct NodeArgs {
```

Comment:
Probably would be useful to also support having anvil read the config from `foundry.toml`. Especially since soon `cast` will also behave this way (https://github.com/gakonst/foundry/pull/1245), so having all 3 tools behave consistently seems ideal 

(Happy to pull all these comments into separate issues if you want, let me know, just wasn't sure of the scope of this PR)